### ISSUE TITLE
[GET] /jobs/username/status

### ISSUE THREAD

**Issue by fetzig** [backmeup/backmeup-prototype#39]
returns:

``` javascript
{
  backupStatus: [
    {
      message: "BackupJob Started",
      type: "info",
      timeStamp: "1349432938168",
      files: [ ]
    },
    ...
```

missing properties:
- backup job id: there is no chance to associate the log entry with a backup job
- category: the log entry should be associated with a category.
  - "started": started job
  - "downloading": downloading from source(s)
  - "uploading": uploading to skin
  - "finished": finished

**Comment by rsimon** [comment]
List of files is now implemented & no longer empty

**Comment by WolfgangEibner** [comment]
Fixed in terms of #39. Maybe modifications needed according to #42.
